src/store/counterData/reducer.js

```diff
@@ -0,0 +1,99 @@
+const initialState = {
+  toDo: [],
+  toDoBoofer: [],
+};
+
+function rootReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'ADD_TODO':
+      return {
+        ...state,
+        toDo: [...state.toDo, action.payload],
+        toDoBoofer: [...state.toDo, action.payload],
+      };
+    case 'REMOVE_TODO':
+      return {
+        ...state,
+        toDo: state.toDo.filter((_, index) => index !== action.payload),
+        toDoBoofer: state.toDo.filter((_, index) => index !== action.payload),
+      };
+    case 'CLEAR_COMPLETED':
+      return {
+        ...state,
+        toDo: state.toDo.filter(item => !item.isDone),
+        toDoBoofer: state.toDo.filter(item => !item.isDone),
+      };
+    case 'READY_TOGLE':
+      return {
+        ...state,
+        toDo: state.toDo.map((element, index) => {
+          if (index === action.payload) {
+            return {
+              name: element.name,
+              isDone: !element.isDone,
+            };
+          }
+          return {
+            name: element.name,
+            isDone: element.isDone,
+          };
+        }),
+        toDoBoofer: state.toDo.map((element, index) => {
```

Comment:
I told you to create each item with **value, id, status**, and remove it only by current id.
Status should be changed only by actions, and will be filtered only one array of items.
Why you ignore that?

remove all cases, and write it correctly.
addToDo, removeToDo, etc.